library/src/test/java/org/apache/apex/malhar/lib/state/managed/ManagedTimeStateImplTest.java

```diff
@@ -81,6 +83,7 @@ public void testAsyncGetFromReaders() throws IOException, ExecutionException, In
     Slice zero = ManagedStateTestUtils.getSliceFor("0");
     long time = System.currentTimeMillis();
 
+    DefaultBucket.setDisableBloomFilterByDefault(true);
```

Comment:
I understand. In that case, I think we should have a unit test which actually executes the code changes done for supporting bloom filter. Probably a test which inserts a fixed number of keys into the store and uses bloom filter to see how much can be fetched without accessing the disk.